apps/aecore/test/aec_mining_tests.erl

```diff
@@ -174,4 +175,16 @@ cleanup(_, PoWMod) ->
             ok
     end.
 
+mine(_, _, 0) ->
```

Comment:
Unused code?

Reply:
Yes.
Also line 118. I understand this was draft code for actually finding nonce that has a solution for updating test.